content/vue/es/get-started.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Empezando"
+tocTitle: "Empezando"
+description: "Configurar Vue Storybook en tu entorno de desarrollo"
+---
+
+# Empezando
+
+Storybook se ejecuta junto con tu aplicación en modo desarrollo. Te ayuda a crear componentes de UI aislados de la lógica y el contexto de su aplicación. Esta edición de Aprende Storybook es para Vue; otras ediciones existen para  [React](/react/es/get-started) y [Angular](/angular/es/get-started).
+
+![Storybook and your app](/storybook-relationship.jpg)
+
+## Configurando Vue Storybook
+
+Tendremos que seguir algunos pasos para configurar el proceso de compilación en su entorno. Para empezar, vamos a querer usar [Vue CLI](https://cli.vuejs.org) para configurar nuestro sistema de compilación. También añadiremos[Storybook](https://storybook.js.org/) y [Jest](https://facebook.github.io/jest/) para probar nuestra aplicación. Vamos a ejecutar los siguientos comandos:
+
+```bash
+# Crea tu aplicación, usando un preset que contiene jest:
+npx -p @vue/cli vue create --preset hichroma/vue-preset-learnstorybook taskbox
+cd taskbox
+
+# Añade Storybook:
+npx -p @storybook/cli sb init
+```
+
+Podemos comprobar rápidamente que los distintos entornos de nuestra aplicación funcionan correctamente:
+
+```bash
+# Ejecuta el corredor de prueba (Jest) en un terminal.
+yarn test:unit
+
+# Inicia el explorador de componentes en el puerto 6006:
+yarn run storybook
+
+# Ejecuta el frontend de la aplicación en el puerto 8080:
+yarn serve
+```
+<div class="aside">
```

Comment:
Why there html here? 😅

Reply:
The original translation has this html. I believe it is meant to provide special formatting when displayed on [https://www.learnstorybook.com/vue/en/get-started](url). I believe that it needs to be here.